src/convertToObject.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyles = {};
+
+  sourceString.split(':')
+    .map((el) => el.trim())
+    .join(':')
+    .split(';')
+    .map((el) => el.trim())
+    .filter((el) => el.length > 0)
+    .forEach((el) => {
+      const keyValue = el.split(':');
+
+      objectOfStyles[keyValue[0]] = keyValue[1];
```

Comment:
de-structuring is very spot-on here
```suggestion
      const [cssName, cssValue] = el.split(':');

      objectOfStyles[cssName] = cssValue;
```